src/presentation/http/router/note.ts

```diff
@@ -97,7 +87,7 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
 
     const noteSettings = await noteService.getNoteSettingsByNoteId(note.id);
 
-    if (noteSettings?.enabled) {
+    if (noteSettings?.enabled === true) {
```

Comment:
Why?

Reply:
Because I've added [strict-boolean-expressions](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md) to the ESLint. 

It's more explicit and makes code more stable preventing some bugs like this (condition always true, even if we have empty object `{}`):

<img width="782" alt="image" src="https://github.com/codex-team/notes.api/assets/3684889/93cd3396-4f50-44c7-8b81-72ab1d0172b6">